Cargo.toml

```diff
@@ -28,11 +28,11 @@ chrono = { version = "0.4", default-features = false, features = ["clock"] }
 nix = "0.26"
 cap-std = "1.0"
 thiserror = "1.0"
-libc = "0.2.145"
+libc = "0.2.146"
 oci-spec = { version = "0.6.0", features = ["runtime"] }
 sha256 = "1.1"
 # TODO: Wait for the release and use the crates.io one.
-libcontainer = { git = "https://github.com/containers/youki", rev = "edd63c84f903aa09510ef758ea6f617e0cb8b7e1" }
+libcontainer = { git = "https://github.com/containers/youki", rev = "3a790dffe9d58f186ead31a1f5e71587ce67124c" }
```

Comment:
Personally, I would like to have a comment there saying what branch is this commit in. It can be as simple as "# main".